tests/codeceptjs/codecept.json

```diff
@@ -7,7 +7,7 @@
   },
   "helpers": {
     "WebDriver": {
-      "url": "http://node:9001/tests/pages/",
+      "url": "http://127.0.0.1:9001/tests/pages/",
```

Comment:
Please do not change these URLs, they need to run in the CI pipelines with Docker.

Reply:
Sorry, this was part of a work-in-progress commit. Fixed now.